internal/hcl/parser.go

```diff
@@ -234,15 +243,33 @@ func LoadParsers(ctx *config.ProjectContext, initialPath string, loader *modules
 
 			var varFiles []string
 			var autoVarFiles []string
+
+			// first remove all the "auto" tfvar files from the list of discovered var files.
+			// These files should not constitute a new "project" as they won't define an
+			// environment but defaults that should be applied across all environments.
 			for _, varFile := range rootPath.TerraformVarFiles {
-				if strings.HasSuffix(strings.TrimSuffix(varFile, ".json"), ".auto.tfvars") {
+				withoutJsonSuffix := strings.TrimSuffix(varFile, ".json")
+
+				if strings.HasSuffix(withoutJsonSuffix, ".auto.tfvars") {
+					autoVarFiles = append(autoVarFiles, varFile)
+					continue
+				}
+
+				if strings.HasSuffix(withoutJsonSuffix, "defaults.tfvars") {
```

Comment:
Should we maintain a list of these? Others I've seen are:
* shared
* global

Although we'd want to exclude the case where they are prefixed with something like `prod-` or `dev-`

Reply:
Yeah probably a good shout, I'll add a list with the ones you've suggested and check they don't have env prefixs